### DISCUSSION TITLE
Fish 550 Discussion Board

### DISCUSSION THREAD

**Discussion by eeholmes** [atsa-es/fish550-2023#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with each other during the FISH 550 Spring 2023 class:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Class webpage: https://atsa-es.github.io/atsa/

**Comment by mdscheuerell** [comment]
Hey @eeholmes, maybe we should do a demo for folks about GitHub Discussions?
